src/FAQ/RestAPI/FAQField.php

```diff
@@ -12,35 +10,35 @@
  */
 class FAQField extends CreatesFields
 {
-
     /**
      * Create an additional field on an array.
-     *
-     * @param WP_Post $post
-     *
-     * @return array
      */
     public function create(WP_Post $post): array
     {
         return array_map(function ($faq) {
             return [
                 'question' => $faq['pdc_faq_question'] ?: '',
-                'answer'   => isset($faq['pdc_faq_answer']) ? apply_filters('the_content', $faq['pdc_faq_answer']) : ''
+                'answer'   => ! empty($faq['pdc_faq_answer']) ? apply_filters('the_content', $this->alterYouTubeEmbedURL($faq['pdc_faq_answer'])) : ''
```

Comment:
Would it be an idea to give this a more generic name? i.e. alterContent or filterContent? Since you could do a lot more with the function?

Reply:
Done.